src/bitcoin-cli.cpp

```diff
@@ -440,6 +440,21 @@ static int CommandLineRPC(int argc, char *argv[])
         const bool fWait = gArgs.GetBoolArg("-rpcwait", false);
         do {
             try {
+                if (method == "getblock") {
+                    if (args.size() >= 1 && args[0][0] == '%') {
```

Comment:
Shouldn't this be `== '@'` or am I missing something?

Reply:
I figured having "@123" exercise the server-side code and "%123" exercise the client-side code would be easier while trying the two approaches out.